pkg/server/testserver_test.go

```diff
@@ -40,18 +35,53 @@ func BenchmarkTestServerStartup(b *testing.B) {
 		b.Run(tc.name, func(b *testing.B) {
 			b.ResetTimer()
 			for i := 0; i < b.N; i++ {
-				s := serverutils.StartServerOnly(b, base.TestServerArgs{
+
+				args := base.TestServerArgs{
 					DefaultTestTenant: tc.tenantOpt,
+					Settings:          cluster.MakeTestingClusterSettings(),
 					Knobs: base.TestingKnobs{
 						SQLEvalContext: &eval.TestingKnobs{
 							// We disable the randomization of some batch sizes to get consistent
 							// results.
 							ForceProductionValues: true,
 						},
 					},
-				})
+				}
+				// Disable leader fortification as it currently causes large variability in runtime.
```

Comment:
is there a way we can force the test to always run with `COCKROACH_INTERNAL_DISABLE_METAMORPHIC_TESTING` set? or something else to make metamorphic constants not get used?

https://github.com/cockroachdb/cockroach/blob/4e70581d869daf562c6433b59ac9a3eb5eabcd25/pkg/util/metamorphic/constants.go#L270-L285

(this seems generally useful for all microbenchmarks):